server/test/unit/server/webservices/test_urls.py

```diff
@@ -4,6 +4,9 @@
 
 
 class TestDjangoPluginsUrls(unittest.TestCase):
```

Comment:
Consider using a single big class for all URLs tests. 1 test per url would be 102 methods on the class. I think having all of those tests in one place is better than spreading them out everywhere. @asmacdo @mhrivnak What do you think?